### PR TITLE
Fix redraw error due to ambiguous comparison

### PR DESCRIPTION
**Pull request by pllim**
This is an addition for ejeschke/ginga#856 to work for me.

### CODE REVIEW

File: ginga/canvas/types/astro.py [hunk 1]
```diff
@@ -1276,7 +1276,7 @@ def draw(self, viewer):
             update = True
 
         cur_limits = viewer.get_limits()
-        if cur_limits != self._cur_limits:
+        if np.any(cur_limits != self._cur_limits):
```
Comment (ejeschke) [inline ginga/canvas/types/astro.py:1279]: How about
```python
if not np.isclose(cur_limits, self._cur_limits):
``` ?

Comment (pllim) [reply ginga/canvas/types/astro.py:1279]: Yes, I think for this purpose, `isclose` would work just the same?

**Comment by pllim** [comment]
If you want to apply `isclose` directory to your PR branch without accepting this PR, that's fine too.

File: ginga/canvas/types/astro.py (hunk 1 above)
Comment (ejeschke) [reply ginga/canvas/types/astro.py:1279]: @pllim, I just understand `np.isclose` better than np.any, but I'm ok with either. You choose, please!

Comment (pllim) [reply ginga/canvas/types/astro.py:1279]: `isclose` is more elegant! Go ahead. 😄

Comment (ejeschke) [reply ginga/canvas/types/astro.py:1279]: I had to use `np.any` with `np.close` so it's a win-win! :smile:

Comment (ejeschke) [reply ginga/canvas/types/astro.py:1279]: I pushed up a commit on the PR.

**Pull request closed without merging**